src/nvim/testdir/test_spell.vim

```diff
@@ -0,0 +1,20 @@
+" Test spell checking
+" TODO: move test58 tests here
+
+if !has('spell')
```

Comment:
I guess change this to `if v:false` for now. Need to think about how to setup spell support for tests.

Reply:
@justinmk 
`if v:false` build failed, can we just return?
````
if !has('spell')
  return
endif
```

Reply:
It would have to be `v:true`, not `v:false`.  `if v:false` will always be false, so we'll never return.